src/database/mod.rs

```diff
@@ -24,3 +24,14 @@ async fn connect_to_database() -> Result<Pool<Postgres>, sqlx::Error> {
 
     Pool::connect_with(options).await
 }
+
+pub async fn perform_migrations(pool: Pool<Postgres>) {
+    let result = migrate!("./migrations").run(&pool).await;
```

Comment:
```suggestion
pub async fn perform_migrations(pool: &Pool<Postgres>) {
    let result = migrate!("./migrations").run(pool).await;
```

This way we don't have to clone it later on.